tooling/ef_tests/state_v2/src/modules/block_runner.rs

```diff
@@ -144,7 +144,7 @@ pub async fn run_test(test: &Test, test_case: &TestCase) -> Result<(), RunnerErr
 
     // 3. Create Blockchain and add block.
 
-    let blockchain = Blockchain::new(store.clone(), BlockchainType::L1, false);
+    let blockchain = Blockchain::new(store.clone(), BlockchainType::L1, false, None);
```

Comment:
As a heads-up, #4639 added a `BlockchainOptions` struct with these parameters, so you might need to put the `fee_vault` inside that.